source/migrating_api.md

```diff
@@ -0,0 +1,450 @@
+  "deployment__encrypted": false,
+  "deployment__uuid": "f739945244514a6bb304dc35d6049880",
+  "tag_string": "my_tag,my_other_tag",
+  "uid": "a4etXeWtqcoodSxLV8a6Uq",
+  "name": "Pathways Initiative",
+  "permissions": [
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pQnggGHaPGGmJtPCbCCVpU/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/add_submissions.json",
+      "label": "Add submissions"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pcn2g8ezroevNsP7CuCkrf/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/change_asset.json",
+      "label": "Edit form"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pVMKQ4bDSn5SCAgefzMRLX/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/change_submissions.json",
+      "label": "Edit submissions"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pwoCoBBLmyWsdF2dXPBaHt/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/delete_submissions.json",
+      "label": "Delete submissions"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pqDGr7M7au4NsFiuKkEeTr/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/manage_asset.json",
+      "label": "Manage project"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/p3dFAXPvHX3ib93twD7rPG/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/validate_submissions.json",
+      "label": "Validate submissions"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pE38Wi89CavBvNHGqQ2WZj/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/view_asset.json",
+      "label": "View form"
+    },
+    {
+      "url": "https://kf.kobotoolbox.org/api/v2/assets/a4etXeWtqcoodSxLV8a6Uq/permission-assignments/pWeJkdjjFk9cQVFvLnr6cS/",
+      "user": "https://kf.kobotoolbox.org/api/v2/users/project_owner.json",
+      "permission": "https://kf.kobotoolbox.org/api/v2/permissions/view_submissions.json",
+      "label": "View submissions"
+    }
+  ]
+}
```

Comment:
Sorry, wrong format for `user` and `permission` properties. 
you should replace `.json` with a `/` instead.